core/src/main/resources/reference.conf

```diff
@@ -109,6 +109,13 @@ avastRabbitMQProducerDefaults {
 
   reportUnroutable = true
 
+  properties {
+    deliveryMode = 2 // 2 is persistent, 1 non-persistent
+    // contentType = ""
+    // contentEncoding = ""
+    // priority = 1
```

Comment:
Why not to support also headers and other properties?

Reply:
Because it wouldn't so easy to implement. We can implement it later if it would be needed (I don't think so).

Reply:
OK